pylint/checkers/format.py

```diff
@@ -168,6 +168,17 @@ def _token_followed_by_eol(tokens, position):
             tokens.type(position+2) == tokenize.NL)
 
 
+def _get_indent_string(line):
+    """Return the indention string of the given line."""
+    result = ''
+    for char in line:
+        if char == ' ' or char == '\t':
```

Comment:
This if statement could be written as `if char in '\t ':` (no need to change if you don't want to though)